### PR TITLE
Fix Haddocks CI by installing secp256k1

### PR DESCRIPTION
**Pull request by amesgen**
Started failing since #3661

### CODE REVIEW

**Comment by Jasagredo** [comment]
I re-added the `nix` label after seeing that it is described to contain everything related to "ci, and also nix"

**Review by Jasagredo — APPROVED** [review]
This looks correct to me as it seems to work on the `build` workflows.

Perhaps we could also use this opportunity to remove the steps related to windows from this workflow as the matrix.os only specifies linux?

**Review by coot — APPROVED** [review]

**Comment by amesgen** [comment]
> Perhaps we could also use this opportunity to remove the steps related to windows from this workflow as the matrix.os only specifies linux?

Sounds good, will do, thanks :+1:

File: .github/workflows/github-page.yml [hunk 1]
```diff
@@ -82,6 +57,22 @@ jobs:
         sudo apt-get -y remove --purge software-properties-common
         sudo apt-get -y autoremove
 
+    - name: Install secp256k1
+      if: matrix.os == 'ubuntu-latest'
```
Comment (amesgen) [inline .github/workflows/github-page.yml:61]: Just noted that this was `ubuntu-20.04` before, which is incorrect in this PR. Now fixed.

**Pull request merged**